examples/kaniko/skaffold.yaml

```diff
@@ -4,8 +4,8 @@ build:
   artifacts:
   - imageName: gcr.io/k8s-skaffold/skaffold-example
   kaniko:
-    gcsBucket: k8s-skaffold
-    pullSecret: ~/secrets/kaniko/kaniko.json
+    gcsBucket: skaffold-debug
```

Comment:
great, glad this bucket is being used, I think I created it to test logs from cloudbuild :) 
Maybe we could create a specific one for integration tests? Or maybe an example specific one so that the example itself is a bit more self-descriptive?

Reply:
Done.